source/features/dim-bots.tsx

```diff
@@ -20,7 +20,7 @@ const commitSelectors = botNames.map(bot => `.commit-author[href$="?author=${bot
 commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic `[bot]` label in author name
 const commitSelector = commitSelectors.join(',');
 
-const prSelectors = botNames.map(bot => `.opened-by [title="pull requests opened by ${bot}"]`);
+const prSelectors = botNames.map(bot => `.opened-by [title="pull requests created by ${bot}"]`);
```

Comment:
```suggestion
const prSelectors = botNames.map(bot => `.opened-by [title*="pull requests created by ${bot}"]`);
```

The `*` is important to have a contains-like comparison

Reply:
Sorry I must have missed it 😅